guides/inventory/overview.md

```diff
@@ -0,0 +1,131 @@
+inventory system using Spree's application-wide configuration.
+
+In your application's `/config/initializers/spree.rb` file, you can add your
+`track_inventory_levels` configuration to the main `Spree::Config` block:
+
+```ruby
+# /config/initializers/spree.rb
+Spree.config do |config|
+  config.track_inventory_levels = false
+end
+```
+
+## Inventory units
+
+A `Spree::InventoryUnit` object is created every time that an item is sold. It
+tracks the state of the sold item. The state could be `on_hand`, `backordered`,
+`shipped`, or `returned`.
+
+Inventory units associate each sold item with many other Solidus models. This
+includes the specific variant that was sold, an order, a line item, and a
+shipment.
+
+<!--For more information, see the [Inventory units][inventory-units] article.-->
+
+[inventory-units]: inventory-units.md
+
+## Stock management
+
+Before a `Spree::InventoryUnit` is created, your store's stock is tracked using
+a number of stock management models. The following sections summarize the
+models and their functions in the stock management system.
+
+### Stock items
+
+On-hand inventory is tracked using the `Spree::StockItem` model. Each
+`Spree::Variant` in a store has a corresponding `Spree::StockItem` object with a
+`count_on_hand` value that represents the number of items you have in stock.
+
+Note that if you have two [stock locations](#stock-locations), there are two
+`Spree::StockItem`s for each variant in your store: one for each
+`Spree::StockLocation`. Each stock item counts the number of items in stock at a
+specific stock location.
+
+<!--For more information, see the [Stock items][stock-items] article.-->
+
+[stock-items]: stock-items.md
+
+### Stock movements
+
+Whenever stock items are sold to customers, added to inventory, or removed from
+inventory, a new `Spree::StockMovement` object is created. The stock movement
+object documents how many items were added or removed.
+
+Each `Spree::StockMovement` corresponds with a `Spree::StockItem` and how much
+the item's `count_on_hand` increases or decreases.
+
+<!--For more information, see the [Stock movements][stock-movements] article.-->
+
+[stock-movements]: stock-movements.md
+
+### Stock locations
+
+A `Spree::StockLocation` represents a location where your inventory is shipped
+from. Each stock location has many `Spree::StockItem`s and
+`Spree::StockMovement`s.
+
+Once a new stock location has been created, a new set of
+[`Spree::StockItem`s](#stock-items) are created for it. The new set represents
+every `Spree::Variant` in your store. 
+
+#### Stock transfers
+
+If you manage multiple stock locations and inventory frequently moves between
+them, you may benefit from the
+[`solidus_stock_transfers`][solidus-stock-transfers] extension. This extension
+adds a user interface for managing transfers in the `solidus_backend`.
+
+[solidus-stock-transfers]: https://github.com/solidusio-contrib/solidus_stock_transfers
+
+### Return authorizations
```

Comment:
It's not immediately clear (at least to me) how this section is related with stock management. I'm not saying it's not related but I think we should specify how here.

We should probably say that the return authorization is something that allows to create `ReturnItems`. Each `ReturnItem` in associated with an inventory unit that can be restocked when order items are received back. You can follow the code [here](https://github.com/solidusio/solidus/blob/9cac52900fadaf79b3157ecd3f0dc2016ae56528/core/app/models/spree/return_item.rb#L81) to see what happens. 

Do you think it could make sense to add these information?

Reply:
Thank you. This prompted me to restructure the documentation around returns. All of the RMA-related documentation I wrote now lives in it's own `returns/` section, and I've added a stub specifically about `Spree::ReturnItem`s to the inventory overview.

There is definitely more to say about RMAs, `ReturnItem`s and other aspects of the returns system. That will have to be added in future PRs.

Reply:
Cool, makes sense!